lib/task_bunny/message.ex

```diff
@@ -1,6 +1,12 @@
 defmodule TaskBunny.Message do
   @moduledoc """
-  Functions to access messages and its meta data.
+  Functions that work on TaskBunny messages.
+
+  It's a semi private module and it is used by Job or Worker module.
```

Comment:
It's => It is

Reason: later in the sentence `it is` is written non contracted.